src/it/banduplicate-classes-fail-when-not-identical/pom.xml

```diff
@@ -0,0 +1,64 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
This integration test ensures we still fail the build when the same class is on the classpath multiple times. In particular, this test uses the new mode (ignoreWhenIdentical=true) but since the duplicate classes differ, the build should still fail.

This ensures the new mode doesn't break existing functionality.